src/website/webview/static/style.css

```diff
@@ -533,14 +533,20 @@ span.hint {
   background: darkgrey;
 }
 
-.green-button {
-  background: green;
+.selected-color {
+  background: var(--selected-suggestion);
+  color: white;
+  cursor: pointer;
+}
+
+.draft-color {
+  background: var(--draft);
   color: white;
   cursor: pointer;
 }
 
-.green-button:hover {
-  background: darkgreen;
+button:hover {
+  filter: brightness(85%);
```

Comment:
This is clever to abstract away, but will probably break down eventually. Let's leave it at that, but it will likely need follow-up work.